apps/devhub/templates/devhub/docs/case-studies-cooliris.html

```diff
@@ -1,110 +0,0 @@
-{% extends "devhub/docs/case-studies.html" %}
-
-{% block title %}{{ docs_page_title('Cooliris Case Study') }}{% endblock %}
-
-{% block content %}
-{{ docs_breadcrumbs(items=[(url('devhub.docs', doc_name='case-studies'), 'Case Studies'),
-                           (None, 'Cooliris')]) }}
-
-{% set link_cooliris = url('addons.detail', 'cooliris') %}
-
-<h2>Case Study: Immersive 3-D Experience with Cooliris</h2>
-<div id="casestudy" class="primary full" role="main">
-{{ sidebar() }}
-
-{# No trans tags because we aren't localizing this yet, per remora comments. #}
-
-<div class="info">
-  <ul class="blocks">
-    <li class="item"><img src="/media/img/docs/case-studies/cooliris-wordmark.png" alt="Immersive 3-D Experience with Cooliris"/></li>
```

Comment:
same with these images.  Looks like a bunch of static assets could go with this patch